javascript/javascript1/week2/candy-helper.js

```diff
@@ -0,0 +1,42 @@
+
+let boughtCandyPrices = []
+function addCandy (candyType, weight){
+    if (candyType === 'sweet'){
+        pricePerGram = 0.5
+    }
+    else if (candyType === 'chocolate'){
+        pricePerGram = 0.7
+    }
+    else if (candyType === 'toffee'){
+        pricePerGram = 1.1
+    }
+    else if (candyType === 'chewing-gum'){
+        pricePerGram = 0.03
+    }else {
+        pricePerGram =0
+    }
```

Comment:
This could be a use case for `switch-case` because all conditionals check the same variable (`candyType`) but using `if-else` is just as good IMO.